go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/sethvargo/go-password v0.2.0
 	github.com/stretchr/testify v1.8.4
-	go.mongodb.org/atlas v0.31.0
+	go.mongodb.org/atlas v0.32.0
```

Comment:
uhm is this upgrade needed by this change? if not I would advise against including it here. We had a dependabot attempt that failed today !1058 and it is being included and fixed by !1041 from @roothorp already, if I am not mistaken.

But maybe it is required.

Reply:
This is #1041 😄

Reply:
update is required. You can see the changes needed in this PR [https://github.com/mongodb/go-client-mongodb-atlas/pull/509](url)